lib/python/ensembl/brc4/runnable/dump_stable_ids.py

```diff
@@ -0,0 +1,420 @@
+#!env python3
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+from os import path
+from typing import Any, List, Dict
+from datetime import datetime
+
+from ensembl.brc4.runnable.core_server import core_server
+
+
+class UnsupportedEvent(ValueError):
+    pass
+
+
+class StableIdEvent:
+    """Represents a stable id event from one gene set version to another one. Various events:
+    - new genes
+    - deleted genes
+    - merged genes (several genes to one)
+    - split genes (one gene to several)
+    - mixed (several genes to several)
+
+    Args:
+    from_list = List of genes the previous gene set
+    to_list = List of genes in the new gene set
+    release = new gene set release name
+    date = date of the new gene set
+
+    Any gene set before 2019-09 is dubbed pre-BRC4.
+
+    Methods:
+    brc_format_1: Draft representation of events for BRC4 v1
+    brc_format_2: Draft representation of events for BRC4 v2
+    add_from: add gene id to the from_list
+    add_to: add gene id to the to_list
+    set_release: set the release
+    set_date: set the release date
+    get_full_release: Retrieve the release name, pre-BRC4 or build
+    get_name: Get the name of the event determined from the from_list and to_list
+    add_pair: Add a pair of gene ids as part of the event
+    add_pairs: Store all the pairs of ids used for this event
+    """
+
+    date_brc4_start = datetime(2019, 9, 1)
```

Comment:
I would declare this as a constant of the module, not an attribute of the class.